backend/src/main/resources/application-prod.properties

```diff
@@ -5,7 +5,7 @@ server.servlet.context-path = /api/v1
 
 server.port=8080
 
-spring.data.neo4j.uri=bolt://localhost:7687
+spring.data.neo4j.uri=neo4j://neo4j:7687
```

Comment:
just a comment, neo4j:// instead of bolt:// simply allows to have a neo4j cluster instead of only a single instance. We won't make use of that, but its good. the neo4j: part is from the docker-compose file (there's a link: part) You already knew this but this is a comment for others if they are wondering.

Reply:
Yes indeed! The link is provided in the dockerfile. So the backend service is linked with the neo4j database service.